frontend/src/Navigation.ts

```diff
@@ -33,7 +33,8 @@ const isNavItem = (extension: Extension): extension is NavExtension => {
 const getAllExtensions: GetAllExtensions = async () => {
   const results: PromiseSettledResult<Extension[]>[] = await Promise.allSettled(
     activePlugins.flatMap(async ({ name: pluginName, pathPrefix = '/api/plugins' }: EnabledPlugin) => {
-      const url = `${pathPrefix}/${pluginName}/plugin-manifest.json`;
+      // TODO: once we are hook use chrome to calculate root
+      const url = `./${pathPrefix}/${pluginName}/plugin-manifest.json`;
```

Comment:
Will this work? 🤔 

Seems odd why we would use a base path at all if this works.

Reply:
It works on deployed applications, locally basePath is not correct for some reason, that's why we are using the chrome's hook to get the proper env (including /beta if on beta env).

Reply:
Then isn't this a moot change?

Reply:
Yeah! You are correct, since we are in hook I can use chrome's hook to calculate the base.